lib/assets/javascripts/chapman_events_feed.js

```diff
@@ -90,14 +102,47 @@
   };
 
   ChapmanEventsFeed.prototype.onFeaturedSuccess = function(data) {
-    this.$featured_element.addClass('contains-featured'); 
-    this.$featured_element.append('<div class="featured-column">' + data + '</div>');
+    $featured_event = $('<div class="featured-event event-card"></div>')
+        .append(this.eventCardDate(data.formatted_date))
+        .append(this.eventCardImg(data))
+        .append(this.eventCardText(data));
+    this.$featured_element.addClass('contains-featured');
+    this.$featured_element.append($featured_event);
   };
 
   ChapmanEventsFeed.prototype.onFeaturedError = function(message) {
     console.log("There's no featured event for the current events feed.");
   };
 
+  ChapmanEventsFeed.prototype.eventCardDate = function(date) {
+    var date_arr = date.split(",");
+    var date_html = [
+      '<div class="event-date"><div class="date-wrapper">',
+        '<div class="left date-column"><p>'+ date_arr[0] +'</p></div>',
+        '<div class="right date-column"><p>'+ date_arr[1] +'</p><p>'+ date_arr[2] +'</p></div>',
+      '</div></div>'
+    ];
+
+    return date_html.join('\n');
+  };
```

Comment:
I think this is ok. Better than straight string concatenation. The alternative using jQuery might be something like:

```
$event_date = $('<div />').addClass('event-date');
$date_wrapper = $('<div />').addClass('date-wrapper');
// create other DOM elements.
$date_wrapper.append($left_date_col).append($right_date_col);
$event_date.append($date_wrapper);
```

In this case, your code is probably more readable and concise.

We should add this topic to our best practices guide.